cupy/linalg/solve.py

```diff
@@ -294,6 +297,79 @@ def inv(a):
     return b
 
 
+def _batched_inv(a):
+
+    assert(a.ndim == 3)
```

Comment:
(not mandatory) For a further compatibility with NumPy, `reshape` or `ascontiguousarray` might add support for `a.ndim > 3` cases.

Reply:
Thanks! will add support for a.ndim > 3 use cases.